### ISSUE TITLE
[Feature Request] $customRoleUser[roleName]  names of people in a custom role.

### ISSUE THREAD

**Issue by CKY-** [crowbartools/Firebot#1616]
**Describe the solution you'd like**
we have 
```
$customRoleUserCount[roleName] 
Get the number of people in a custom role.
```
to be used with loops but there is no way to connect them 

```
$customRoleUser[roleName]  names of people in a custom role.
```
```
$customRoleUser[roleName, index]  name of person in a custom role at the specified index.
```

**Comment by zunderscore** [comment]
Implemented the first part in #1704. Didn't add the one with the `index`. Instead, using this within the new `$arrayElement` variable should suffice.

**Comment by SReject** [comment]
Resolved with v5.52.0
